adafruit_lsm303dlh_mag.py

```diff
@@ -29,13 +29,14 @@
   https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
 
+from typing import Tuple
```

Comment:
Sorry, that wasn't clear of me.  Things should be wrapped in a `try`/`except` block, with the first import being from the `typing` module.  Since it isn't available on CircuitPython, it fails and skips importing everything else in that block.  Since CircuitPython discards annotations, it's fine if those things that are skipped since they aren't parsed at runtime.  So your typing block should look like this:

```suggestion
try:
    from typing import Tuple
    from busio import I2C
except ImportError:
    pass
```

Reply:
I would then remove the remnants of that original `try`/`except` block you added since it's now in the one here.